### PR TITLE
Fb 73 ability to auth as a user from the command line 2

### PR DESCRIPTION
**Pull request by JakeGinnivan**
Add `fb login`, `fb account` and `fb select-organization` commands to CLI to authenticate with the FeatureBoard service via OAuth

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,3 +13,5 @@ To run the CLI locally run the following command (with the arguments you want to
 ```bash
 pnpm cli
 ```
+
+`pnpm cli login` will login to the dev app registration by default. If you want to login to production FeatureBoard
```
Comment (emlyn-rice) [inline CONTRIBUTING.md:17]: What do I do to login to prod ????

File: apps/cli/project.json [hunk 2]
```diff
@@ -4,14 +4,25 @@
     "sourceRoot": "apps/cli/src",
     "projectType": "application",
     "targets": {
+        "copy-templates": {
+            "executor": "nx:run-commands",
+            "options": {
+                "commands": [
+                    {
+                        "command": "cpy ./libs/code-generator/src/lib/templates/dotnet-api/* ./apps/cli/dist/templates/dotnet-api/ --flat"
```
Comment (emlyn-rice) [inline apps/cli/project.json]: Can we just copy ./libs/code-generator/src/lib/templates/*
And that way we don't need to maintain this?

File: apps/cli/src/commands/code-gen.ts [hunk 3]
```diff
@@ -69,66 +75,76 @@ export function codeGenCommand() {
...
-                        name: 'bearerToken',
-                        message: `Enter your featureboard bearer token:`,
-                        validate: (x) => !!x,
-                    })
-
-                    if (!('bearerToken' in promptResult)) {
+                if (!options.featureBoardApiKey && !options.nonInteractive) {
+                    const token = await getValidToken()
+                    if (!token) {
                         return
```
Comment (emlyn-rice) [inline apps/cli/src/commands/code-gen.ts:108]: Shouldn't there be a message to the user here?

File: CONTRIBUTING.md (hunk 1 above)
Comment (JakeGinnivan) [reply CONTRIBUTING.md:17]: pnpm cli-prod login

File: apps/cli/project.json (hunk 2 above)
Comment (JakeGinnivan) [reply apps/cli/project.json]: I can move the typescript files up, but I didn't want to copy the typescript files in that folder.

File: apps/cli/src/commands/code-gen.ts (hunk 3 above)
Comment (JakeGinnivan) [reply apps/cli/src/commands/code-gen.ts:108]: The message will be printed by that function itself

File: apps/cli/project.json (hunk 2 above)
Comment (JakeGinnivan) [reply apps/cli/project.json]: Done

**Pull request merged**